### PR TITLE
Install @ember/legacy-built-in-components

### PR DESCRIPTION
**Pull request by charlesfries**
Fixes this deprecation warning:

```
DEPRECATION: Using Ember.TextArea or importing from 'TextArea' has been deprecated, install the `@ember/legacy-built-in-components` addon and use `import { TextArea } from '@ember/legacy-built-in-components';` instead [deprecation id: ember.built-in-components.import] See https://deprecations.emberjs.com/v3.x#toc_ember-built-in-components-import for more details.
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "ember-modifier": "^2.1.1"
   },
   "devDependencies": {
+    "@ember/legacy-built-in-components": "^0.2.0",
```
Comment (SergeAstapov) [inline package.json:36]: @charlesfries this should be in `dependencies` as it's needed at run time when this addon is installed into host app

Comment (charlesfries) [reply package.json]: Fixed

**Comment by SergeAstapov** [comment]
@charlesfries see my comment [in #489](https://github.com/adopted-ember-addons/ember-keyboard/pull/489#issuecomment-968399835).

TL;DR `@ember/legacy-built-in-components` does not support Node.js 10 and `ember-keyboard` still does.
